### PR TITLE
Split large files

### PR DESCRIPTION
**Pull request by daisieh**
Run `split_large_bitstream.py` to chunk a large file from the assetstore and upload the chunks to the dryad-web-assets bucket. Run `remove_split_bitstreams.py` to remove the chunks when finished.

### CODE REVIEW

File: remove_split_bitstreams.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        raw = raw_input('Enter the bitstream ID or URL: ')
+        for matcher in matchers:
+            matched_bitstream_id = re.match(matcher, raw)
+            if matched_bitstream_id is not None:
+                break
+    return int(matched_bitstream_id.group(1))
+
+def main():
+    bitstream_id = None
+    if len(sys.argv) == 3:
```
Comment (ryscher) [inline remove_split_bitstreams.py]: should this be "== 2" ? I don't see another argument. Please document the arguments in the comments at the beginning of the script.

**Comment by ryscher** [comment]
won't run unless ASSETSTORE_BUCKET is set... should this be added to the vagrant config?

**Comment by daisieh** [comment]
No, because this script is meant to be run on any machine, not just vagrant-based ones. Specifically, the FTP server.

File: split_large_bitstream.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import os
+import sys
+import json
+import tempfile
+import subprocess
+from sql_utils import dict_from_query, sql_query
+
+WEB_ASSETS_BUCKET = "dryad-web-assets/temporary"
+ASSETSTORE_BUCKET = os.environ['ASSETSTORE_BUCKET']
+TEMP_DIR = '/transfer-complete/temp'
```
Comment (ryscher) [inline split_large_bitstream.py]: This directory won't exist on any machine that wasn't specifically set up for this script. Any reason why we cant use /tmp ?

Comment (daisieh) [reply split_large_bitstream.py]: Yes, these files are generally very very large and won't necessarily be able to be stored in /tmp. On the FTP server, /transfer-complete is an extra mount with 1000 GB of space.

Comment (ryscher) [reply split_large_bitstream.py]: Ok, then add that note to the script comments and this can be merged.

**Pull request merged**